src/content/tools/pub/publishing.md

```diff
@@ -23,7 +23,7 @@ Watch the following video for an overview of building and publishing packages.
   {{yt.set}}>
 </iframe>
 
-## Publishing is forever
+## Publish is forever
```

Comment:
This sounds strange to me compared to the original. Any particular reason for this change?

Reply:
Missed how this change would look. We're trying to remove gerunds from headings.